packages/netstring/package.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "@endo/netstring",
+  "version": "0.1.0",
+  "description": "Description forthcoming.",
+  "keywords": [],
+  "author": "Endo contributors",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/endojs/endo/tree/master/packages/netstring#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/endojs/endo.git"
+  },
+  "bugs": {
+    "url": "https://github.com/endojs/endo/issues"
+  },
+  "type": "module",
+  "parsers": {
+    "js": "mjs"
+  },
+  "main": "./dist/netstring.cjs",
+  "module": "./index.js",
+  "browser": "./dist/netstring.umd.js",
+  "unpkg": "./dist/netstring.umd.js",
+  "types": "./index.d.ts",
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "import": "./index.js",
+      "require": "./dist/netstring.cjs",
+      "browser": "./dist/netstring.umd.js"
+    },
+    "./reader": {
+      "import": "./reader.js",
+      "require": "./dist/netstring-reader.cjs",
+      "browser": "./dist/netstring-reader.umd.js"
+    },
+    "./writer": {
+      "import": "./writer.js",
+      "require": "./dist/netstring-writer.cjs",
+      "browser": "./dist/netstring-writer.umd.js"
+    }
+  },
+  "scripts": {
+    "build": "rollup --config rollup.config.js",
+    "clean": "rm -rf dist",
+    "cover": "nyc ava",
+    "lint": "yarn lint:types && yarn lint:js",
+    "lint-fix": "eslint --fix '*.js' '**/*.js' 'test*/**.js'",
+    "lint:js": "eslint '*.js' '**/*.js' 'test*/**.js'",
+    "lint:types": "tsc --build jsconfig.json",
+    "prepublish": "yarn clean && yarn build",
+    "test": "ava"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@rollup/plugin-commonjs": "^13.0.0",
+    "@rollup/plugin-node-resolve": "^6.1.0",
+    "ava": "^3.12.1",
+    "babel-eslint": "^10.0.3",
+    "eslint": "^7.23.0",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-config-prettier": "^6.9.0",
+    "eslint-plugin-eslint-comments": "^3.1.2",
+    "eslint-plugin-import": "^2.19.1",
+    "eslint-plugin-prettier": "^3.1.2",
+    "nyc": "^15.1.0",
+    "prettier": "^1.19.1",
+    "rollup": "1.31.0",
+    "rollup-plugin-terser": "^5.1.3",
+    "typescript": "^4.0.5"
```

Comment:
why an old typescript? I see "TypeScript 4.2 is now available"

Reply:
https://github.com/endojs/endo/issues/695